parity/informant.rs

```diff
@@ -373,7 +373,8 @@ impl ChainNotify for Informant<FullNodeInformantData> {
 					Colour::White.bold().paint(format!("{}", header_view.hash())),
 					Colour::Yellow.bold().paint(format!("{}", block.transactions_count())),
 					Colour::Yellow.bold().paint(format!("{:.2}", header_view.gas_used().low_u64() as f32 / 1000000f32)),
-					Colour::Purple.bold().paint(format!("{:.2}", duration as f32 / 1000000f32)),
+					// this will not provide floating point precision because `as_milliseconds` returns u64
+					Colour::Purple.bold().paint(format!("{:.2}", duration.as_milliseconds())),
```

Comment:
This can be changed provide the same behaviour as before but since we already had a method for specifically implemented for `Duration` I used that one instead!

Reply:
Since it will always be `.00` I think we should remove the `{:.2}` from the format string. I don't think we care about nano precision wrt time it takes to import a block so I guess it's OK to leave it as millis.

Reply:
Yepp, I agree